drivers/network/tcpip/ip/transport/tcp/tcp.c

```diff
@@ -410,7 +410,16 @@ NTSTATUS TCPConnect
     Status = TCPTranslateError(LibTCPConnect(Connection,
                                                 &connaddr,
                                                 RemotePort));
-
+    if (!NT_SUCCESS(Status)) // Status != STATUS_SUCCESS && Status != STATUS_PENDING
```

Comment:
I don't think you need the comment -- NT_SUCCESS is the standard pattern for this (see my other comment).

Reply:
removed